File: easy/7_factorial_number/factorial_number.go
package main

import "fmt"

// Find factorial of a number

func factorial(num int) int {
	if num == 0 {
		return 1
	}
	return num * factorial(num-1)
}

func main() {
	fmt.Println(factorial(5))
	fmt.Println(factorial(10))
	fmt.Println(factorial(15))
}

File: easy/7_factorial_number/factorial_number.js
// Find factorial of a number

function factorial(num) {
    if (num === 0) {
        return 1;
    }
    return num * factorial(num - 1);
}

console.log(factorial(5));
console.log(factorial(10));
console.log(factorial(15));

module.exports = factorial;